modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectValidation/ModalAddObjectValidation.tsx

```diff
@@ -36,13 +37,23 @@ const initialValues: Partial<ObjectValidation> = {
 export function ModalAddObjectValidation({
 	apiURL,
 	objectValidationRuleEngines,
+	scriptManagementEnabled,
 	setShowAddObjectRelationshipModal,
 }: ModalAddObjectValidationProps) {
 	const [error, setError] = useState<string>('');
 	const {observer, onClose} = useModal({
 		onClose: () => setShowAddObjectRelationshipModal(false),
 	});
 
+	let newObjectValidationRuleEngines = objectValidationRuleEngines;
```

Comment:
same comment from [here](https://github.com/liferay-objects/liferay-portal/pull/3129#discussion_r1520211272)

but if you think this won't cause a problem or a bug, you can leave the way it is!